exercises/concept/logs-logs-logs/logs_logs_logs_test.go

```diff
@@ -121,6 +121,24 @@ func TestWithinLimit(t *testing.T) {
 			limit: 8,
 			want:  false,
 		},
+		{
+			name:  "exact limit",
+			log:   "exercism🔍",
+			limit: 9,
+			want:  true,
+		},
+		{
+			name:  "under limit",
+			log:   "exercism🔍",
+			limit: 10,
+			want:  true,
+		},
+		{
+			name:  "exact limit",
```

Comment:
```suggestion
			name:  "over limit",
```

I think here you meant this.

Reply:
Oh thanks

Reply:
Feel free to make a new commit incorporating this change.

Reply:
Done!
Thanks for the feedback again!